src/modules/Dropdown/Dropdown.js

```diff
@@ -1152,7 +1152,7 @@ export default class Dropdown extends Component {
 
     return _.map(options, (opt, i) => (
       <DropdownItem
-        key={`${opt.value}-${i}`}
+        key={opt.key || `${opt.value}-${i}`}
```

Comment:
Again here, we should add the `create` method to the DropdownItem.  We have a factory creator for this:

**DropdownItem.js**
```jsx
Dropdown.create = createShorthandFactory(Dropdown, val => ({ text: val, value: val }))
```

Then:
```jsx
_.map(options, opt => Dropdown.create(opt)) // PLUS all the other props noted here...
```

Tests:
**DropdownItem-test.js**
```jsx
common.implementsCreateMethod(Dropdown)
```